docs/Plugins.rst

```diff
@@ -289,6 +289,12 @@ showmood
 ========
 Shows all items needed for the currently active strange mood.
 
+spectate
```

Comment:
Adding `.. _spectate:` above the header (like showmood has) might clear up the doc build errors.